client/src/components/History/CurrentHistory/HistoryNavigation.vue

```diff
@@ -73,6 +73,11 @@
                         <span v-localize>Extract Workflow</span>
                     </b-dropdown-item>
 
+                    <b-dropdown-item v-if="!currentUser.isAnonymous" @click="redirect(invocationsLink)">
```

Comment:
Can we only show this, if there are actually invocations in the history?

Reply:
Seems like people might want to know explicitly they haven't run workflow invocations but I can do this... I think the thing to do here is add a HEAD route for /api/invocations to get ``total_matches`` for the query without making the query. I don't want to do that in the old framework though - so I will close this and reopen a new PR based on my work to port more of the workflows API to the newer framework.

Reply:
It's tricky though, we'd have to poll this, because it would certainly be worse to say there are no invocations when there are invocations. I think it's alright to always show this option in a first step if the empty table message said "No workflow invocations in this history"

Reply:
It was just an idea, no need to close it :)